### ISSUE TITLE
Can I use this in flutter app?

### ISSUE THREAD

**Issue by creativekdev** [ChaoningZhang/MobileSAM#84]
I am developing flutter plugin that can use this model with ONNX. I am stuck converting mobile_sam package of python to Kotlin and Swift Side. How can I use the mobile_sam package on Native mobile code side?
![image](https://github.com/ChaoningZhang/MobileSAM/assets/127048539/f0cfb8c6-e7f0-4f6d-b88d-d3c1cdcbbc96)

**Comment by ryouchinsa** [comment]
Here is the [Segment Anything CPP Wrapper for macOS](https://github.com/ryouchinsa/sam-cpp-macos/tree/master).
This code is originated from [Segment Anything CPP Wrapper](https://github.com/dinglufe/segment-anything-cpp-wrapper) and implemented on macOS app [RectLabel](https://rectlabel.com). We customized the original code so that getMask() uses the previous mask result called as low_res_logits and retain the previous mask array for undo/redo actions. 

We hope this macOS version would help to develop the iOS version.
Please let us know your opinion.

![sam_polygon](https://github.com/ryouchinsa/sam-cpp-macos/assets/1954306/4640e139-c533-4b8c-b27b-e02a401b9bbd)

**Comment by zgdaaa** [comment]
> 我正在开发 Flutter 插件，以便将此模型与 ONNX 结合使用。我在将 Python 的 mobile_sam 包转换为 Kotlin 和 Swift 代码时遇到了困难。如何在原生移动代码端使用 mobile_sam 包？ ![image](https://github.com/ChaoningZhang/MobileSAM/assets/127048539/f0cfb8c6-e7f0-4f6d-b88d-d3c1cdcbbc96)

Hello, I also want to use this model on Flutter now. Have you successfully implemented it?
